async_fuse/src/protocol.rs

```diff
@@ -478,8 +478,15 @@ pub enum FuseNotifyCode {
     FUSE_NOTIFY_CODE_MAX,
 }
 
-// The read buffer is required to be at least 8k, but may be much larger
-pub const FUSE_MIN_READ_BUFFER: usize = 8192;
+#[allow(dead_code)]
+pub mod fuse_configs {
```

Comment:
how about group more related const variables and use a more meaningful mod name?

Reply:
done.